src/errorpage.cc

```diff
@@ -206,6 +207,16 @@ static MemBuf error_stylesheet;
 static const char *errorFindHardText(err_type type);
 static IOCB errorSendComplete;
 
+/// \ingroup ErrorPageInternal
+static const char *
+FindDefaultTemplate(const err_type type)
```

Comment:
The PR has two nearly identical functions that find error text by err_type. Can we merge them into one FindHardCodedTemplate(), accepting the (templated) container parameter (one of the two std::arrays) as the second argument? I think it may even improve caller code comprehension because it will emphasize which array is used (without the need to remember which function searches which array).

Reply:
We can merge them. I was thinking about that but tried to minimize changes keeping the old errorFindHardText() function.